### DISCUSSION TITLE
[GitHub Pages] My `*.md` files in the root folder dont parse markdown to html.

### DISCUSSION THREAD

**Discussion by oxmhpark** [jekyll/jekyll#9281] [Q&A] [answered]
1. Target `*.md` files are located in the root folder. (index.md, about.md, archives.md, ...)
 2. `_config.yml` file contains no `include` section.
 3. These files are rendered well with desired layout.
 4. But their {{ content }} are shown as plain text. (reveal markdown codes)

Is any setup required?

**Comment by ashmaroli** [comment, marked as answer]
Since you haven't posted a link to your site's repository, I assumed the repo to be https://github.com/oxmhpark/www.exonomicon.net.
The reason you don't see Markdown rendered there is because your `page` layout doesn't have a `{{ content }}` instead there is a 
`{{ post.content }}` injected via an include file.

`{{ page.content }}` is not *identical* to `{{ content }}`.

**Reply by oxmhpark** [reply]
Thank you! I rewrote my templates.
